tests/endtoend/timer_functions/timer_func/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "name": "mytimer",
+      "type": "timerTrigger",
+      "direction": "in",
+      "schedule": "*/2 * * * * *",
+      "runOnStartup": "true"
```

Comment:
Does this property expect a bool or a string?

Reply:
bool. updated